brd-ios/breadwallet/src/ViewControllers/VerifyPinViewController.swift

```diff
@@ -98,45 +95,33 @@ class VerifyPinViewController: UIViewController, ContentBoxPresenter {
 
     private func addConstraints() {
         contentBox.constrain([
+            contentBox.topAnchor.constraint(equalTo: view.topAnchor, constant: C.padding[20]),
             contentBox.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            contentBox.bottomAnchor.constraint(equalTo: pinPad.view.topAnchor, constant: -C.padding[12]),
             contentBox.widthAnchor.constraint(equalToConstant: 256.0) ])
         titleLabel.constrainTopCorners(sidePadding: C.padding[2], topPadding: C.padding[2])
         body.constrain([
             body.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
-            body.topAnchor.constraint(equalTo: titleLabel.bottomAnchor),
-            body.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor) ])
+            body.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: C.padding[2]),
+            body.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor)])
         pinView.constrain([
-            pinView.topAnchor.constraint(equalTo: body.bottomAnchor, constant: C.padding[2]),
+            pinView.topAnchor.constraint(equalTo: body.bottomAnchor, constant: C.padding[10]),
```

Comment:
Are you sure this works on both small and bug devices?

Reply:
yes, I also tested it on iPhone SE and it’s okay.